src/App.css

```diff
@@ -0,0 +1,53 @@
+
+*{
+margin: 0;
+    padding: 0;
+    font-family:Arial, sans-serif;
+    --food-list-button-color : green;
+    --food-list-button-hover-color : red;
+}
+.flex{
+    display: flex;
+}
+.inline{
+    display: inline;
+}
+.margin-2{
```

Comment:
.margin-2 doesn't signifies the unit like rem or em or px or %, be clear with it